Maven Artifact is broken

Just a heads up, the maven artifact is broken. 

The cause of this has nothing to do with your code but rather that one of the dependencies, [Darcula](https://github.com/bulenkov/Darcula), is hosted on [spring.io](https://spring.io/) and spring has recently implemented some permission changes as stated in [this-Notice of Permissions Changes to repo.spring.io, Fall and Winter, 2020-](https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020) article.

Everything works fine if you get Darcula from source and build UiBooster from source using that

Here is the exact error for others
....................................................

[ERROR] Failed to execute goal on project: Could not resolve dependencies for project

Failed to collect dependencies at com.github.Milchreis:UiBooster:jar:1.9.0 -> com.bulenkov:darcula:jar:2018.2: Failed to read artifact descriptor for com.bulenkov:darcula:jar:2018.2

Could not transfer artifact com.bulenkov:darcula:pom:2018.2 from/to spring (https://repo.spring.io/plugins-release/): Authentication failed for https://repo.spring.io/plugins-release/com/bulenkov/darcula/2018.2/darcula-2018.2.pom 401 Unauthorized

--- a/pom.xml
+++ b/pom.xml
@@ -24,7 +24,7 @@
         </dependency>
 
         <dependency>
-            <groupId>com.bulenkov</groupId>
+            <groupId>com.github.ice1k</groupId>
             <artifactId>darcula</artifactId>
             <version>2018.2</version>
         </dependency>
@@ -39,9 +39,8 @@
 
     <repositories>
         <repository>
-            <id>bintray-ice1000-ice1000</id>
-            <name>bintray</name>
-            <url>https://dl.bintray.com/ice1000/ice1000</url>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
         </repository>
     </repositories>
 

--- a/pom.xml
+++ b/pom.xml
@@ -39,9 +39,9 @@
 
     <repositories>
         <repository>
-            <id>spring</id>
-            <name>spring</name>
-            <url>https://repo.spring.io/plugins-release/</url>
+            <id>bintray-ice1000-ice1000</id>
+            <name>bintray</name>
+            <url>https://dl.bintray.com/ice1000/ice1000</url>
         </repository>
     </repositories>
 

